.travis.yml

```diff
@@ -10,25 +10,15 @@ cache:
 
 language: scala
 
-scala:
-   - 2.11.8
-
 jdk:
    - oraclejdk8
 
-env:
-  global:
-    - SCALA_VERSION=2.11.8
-
 matrix:
   include:
-    - env: PYSPARK_PYTHON=python2 SPARK_VERSION=2.3.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python2 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.3.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-    - env: PYSPARK_PYTHON=python3 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
-
-before_install:
- - ./bin/download_travis_dependencies.sh
+    - env: PYSPARK_PYTHON=python2 SCALA_VERSION=2.11.8 SPARK_VERSION=2.4.4 SPARK_BUILD="spark-${SPARK_VERSION}-bin-hadoop2.7"
```

Comment:
I think it's OK if we don't bother building against 2.4 as well.  The current version should continue to work, and if it breaks, we can always create a 2.4-compatible branch off of an old commit.

Reply:
Fair enough; I'm OK with keeping it for now, but if we have to do a lot of work to maintain 2.4 compatibility, we'd probably end up dropping 2.4 support.